www/docs/en/dev/config_ref/images.md

```diff
@@ -34,15 +31,14 @@ If you do not specify an icon then the Apache Cordova logo is used.
 
         <icon src="res/ios/icon.png" platform="ios" width="57" height="57" density="mdpi" />
 
-src: (required) specifies the location of the image file, relative to your project directory
+Attribute     |      Requirement       |    Description
```

Comment:
The requirement information should be listed in the description column. See the example here: https://github.com/apache/cordova-docs/blob/master/doc/docs-STYLEGUIDE.md#xml-references